vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailViewModel.kt

```diff
@@ -181,7 +181,9 @@ class RoomDetailViewModel @AssistedInject constructor(
         observePowerLevel()
         updateShowDialerOptionState()
         room.getRoomSummaryLive()
-        room.markAsRead(ReadService.MarkAsReadParams.READ_RECEIPT, NoOpMatrixCallback())
+        viewModelScope.launch {
+            room.markAsRead(ReadService.MarkAsReadParams.READ_RECEIPT)
```

Comment:
The several call should be protected by try catch block I think

Reply:
Ah you're right. I'll add that.